### PR TITLE
Suppress `filemtime()` warnings

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

### CODE REVIEW

File: src/FileHelper.php [hunk 1]
```diff
@@ -576,13 +576,13 @@ public static function lastModifiedTime(string|RecursiveDirectoryIterator ...$pa
...
+     * @see https://www.php.net/manual/function.filemtime.php
+     */
     private static function modifiedTime(string $path): ?int
     {
-        if (false !== $timestamp = filemtime($path)) {
-            return $timestamp;
-        }
-
-        return null;
+        $timestamp = @filemtime($path);
```
Comment (samdark) [inline src/FileHelper.php:584]: It is better to be consistent with https://github.com/yiisoft/files/blob/master/src/FileHelper.php#L71 error handling.

Comment (vjik) [reply src/FileHelper.php:584]: `lastModifiedTime()` and this method returns null when it was not possible to get modified time. So we shouldn't throw exception on fail, isn't we?

Comment (samdark) [reply src/FileHelper.php:584]: Yes. I mean, we can either suppress the error with `@` or handle it as in the code I've linked. I prefer handling it.

Comment (vjik) [reply src/FileHelper.php:584]: Set/unset handler needs to done for each of files, there may be many of them.

Comment (samdark) [reply src/FileHelper.php:584]: OK. Let's leave it like that for now.

**Review by samdark — APPROVED** [review]

**Pull request merged**